onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -141,7 +144,9 @@ std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_Nuphar
 #endif  // _MSC_VER
 
 namespace onnxruntime {
+#ifdef ENABLE_TRAINING
 using namespace training;
+#endif
```

Comment:
There are some refactoring done by Edward 
[https://github.com/microsoft/onnxruntime/pull/3214](https://github.com/microsoft/onnxruntime/pull/3214)

Please sync with him on the plan

Reply:
I wonder if training related changes in this file can be refactor into another file... 

This would make the co-development cleaner

---
In reply to: [393961939](https://github.com/microsoft/onnxruntime/pull/3235#discussion_r393961939) [](ancestors = 393961939)